bin/generateArgTypes.ts

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node --experimental-loader esbuild-node-loader
```

Comment:
Any reason why BTW? I prefer CamelCased (like in code) especially to match component names.

Reply:
Yeah, I used to like camel cased too because it matches component names, but I am not using default export any more most of the time and components export more than one thing, so there is no matching, also I don't encourage putting a component per module.

Additionally it avoids case-sensitivity file system/git issues you are probably aware of